test/data/voce_ea_cs.toml

```diff
@@ -0,0 +1,123 @@
+#The below show all of the options available and their default values
+#Although, it should be noted that the BCs options have no default values
+#and require you to input ones that are appropriate for your problem.
+#Also while the below is indented to make things easier to read the parser doesn't care.
+#More information on TOML files can be found at: https://en.wikipedia.org/wiki/TOML
+#and https://github.com/toml-lang/toml/blob/master/README.md 
+[Properties]
+    temperature = 298
+    #The below informs us about the material properties to use
+    [Properties.Matl_Props]
+        floc = "props_cp_voce.txt"
+        num_props = 17
+    #These options tell inform the program about the state variables
+    [Properties.State_Vars]
+        floc = "state_cp_voce.txt"
+        num_vars = 24
+    #These options are only used in xtal plasticity problems
+    [Properties.Grain]
+        ori_state_var_loc = 9
+        ori_stride = 4
+        #The following options are available for orientation type: euler, quat/quaternion, or custom.
+        #If one of these options is not provided the program will exit early.
+        ori_type = "quat"
+        num_grains = 500
+        ori_floc = "voce_quats.ori"
+        grain_floc = "grains.txt"
+#All of these options are required
+#If they are not provided the program will exit early
+[BCs]
+    # Optional - tells the program that we'll have changing BCs
+    changing_ess_bcs = false
+    # Optional - tells the program if we want a constant strain rates rather than
+    # constant velocity boundary conditions
+    constant_strain_rate = true
+    #essential BC ids for the whole boundary
+    essential_ids = [1, 2, 3, 4]
+    #component combo (x,y,z = -1, x = 1, y = 2, z = 3, xy = 4, yz = 5, xz = 6, free = 0)
+    essential_comps = [-3, -1, -2, -3]
```

Comment:
What are the negative values for here? Does it relate to the constant strain rate option?

Reply:
Yes, the negatives here signify that the essential boundary condition should for a given boundary should have the velocity gradient / constant strain rate BC applied rather than the applied velocity BC.

Reply:
Went through and updated all the test options files so this outdated info should no longer be in there